.flake8

```diff
@@ -328,3 +326,10 @@ rst-roles =
 
 # wemake-python-styleguide
 show-source = true
+
+# flake8-quotes (pulled in by wemake style guide)
```

Comment:
Maybe we could follow the same commenting pattern as with the pytest-style plugin above? A line with the plugin name then rules that specific options influence. We probably shouldn't send people off to the docs to try to figure out this connection.

Reply:
I added the we-make thing because the flake8 extension is not otherwise configured and someone might try to remove it w/o knowing the connection to wemake

Reply:
Note that several sections above are also coming from WPS. So we may want to do something about that in a more consistent manner.